nixos/tests/all-tests.nix

```diff
@@ -138,8 +138,8 @@ in
   gitolite = handleTest ./gitolite.nix {};
   gitolite-fcgiwrap = handleTest ./gitolite-fcgiwrap.nix {};
   glusterfs = handleTest ./glusterfs.nix {};
-  gnome3 = handleTest ./gnome3.nix {};
-  gnome3-xorg = handleTest ./gnome3-xorg.nix {};
+  gnome = handleTest ./gnome.nix {};
```

Comment:
Does the test name need to be updated in Hydra job? Or is `nixos/release-combined.nix` enough?

Reply:
This should be fine (as I see you updated the `tested` job, too).  It will be like removing the gnome3 job and adding a gnome job; I don't think there's any way around that.  Well, overall I rather see more potential costs from these mass "renamings".

Reply:
What kinds of costs? We do have aliases.

Reply:
Say, various kinds of merge conflicts.